pages/blog/socketio-automatic-docs.md

```diff
@@ -0,0 +1,555 @@
+---
+title: Flask-SocketIO Automated Documentation and Validation
+date: 2023-01-26T06:00:00+01:00
```

Comment:
```suggestion
date: 2023-03-02T06:00:00+01:00
```

@daler-rahimov sorry for delay with releasing the article, please update the date and we will release and promote it this week